src/screens/HomeScreen.tsx

```diff
@@ -7,14 +7,27 @@ import { TilesList } from '../components/tiles/TilesList';
 import { ScreenSurface } from '../components/ScreenSurface';
 import { HomeStackScreenProps } from '../navigators/types';
 import { useJoinCircles } from '../hooks';
+import { refreshNotifier } from '../common/RefreshNotifier';
+import { HeaderLeftRefreshButton } from '../components/HeaderLeftRefreshButton';
 
 export const HomeScreen = (navProps: HomeStackScreenProps<'Home'>) => {
   const { isLoading: loadingAccount } = useActiveAccount();
   const { isInitialLoading: loadingAppConfig, data: appConfig } =
     useAppConfig();
   const { isInitialLoading: loadingJoinCircles } = useJoinCircles();
+  const [refreshing, setRefreshing] = useState(false);
   const { navigation } = navProps;
   const customHeaderTitle = appConfig?.homeTab?.screenHeader?.title;
+  const headerRefreshEnabled = appConfig?.homeTab?.screenHeader?.enableRefresh;
```

Comment:
It's important to me that for apps that _don't_ want/need this refresh, there's hardly any cost.  The current cost is the additional useEffect & useCallback quick no-op invocations, which I'm thinking is a solid first pass to see how/where this is utilized.